test/decorators.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const { clone } = require('@metarhia/common');
+const metatests = require('metatests');
+
+const { Flags } = require('../lib/decorators').classes;
+
+const {
+  default: def,
+  fs: { load },
+} = require('..');
+
+const { options, config } = clone(def);
+
+metatests.test('Decorators / Flags', async test => {
+  let schema;
+
+  try {
```

Comment:
You can remove `try {} catch () {}` because `metatests` now handles aync functions.